test/crash/src/test/java/io/narayana/lra/arquillian/LRACoordinatorRecoveryIT.java

```diff
@@ -147,60 +148,59 @@ public static WebArchive deploy() {
     }
 
     @Test
-    public void lraCoordinatorShortTimeoutLRA(@ArquillianResource @OperateOnDeployment(LRA_PARTICIPANT_DEPLOYMENT_QUALIFIER) URL baseURL)
+    public void lraCoordinatorShortTimeoutLRAwithParticipant(
+            @ArquillianResource @OperateOnDeployment(LRA_PARTICIPANT_DEPLOYMENT_QUALIFIER) URL baseURL)
```

Comment:
This looks like formatting change only so should be in a separate commit I think? There seems other formatting changes too (removed lines, comment line length) that should be in the separate commit too please